### ISSUE TITLE
learning rate schedule questions

### ISSUE THREAD

**Issue by kunhe** [daijifeng001/R-FCN#31]
First off, thanks a lot for the great work and code release. For Lack of a better place to ask, these are more of implementation questions than actual issues. My apologies!

1. The NIPS paper states (Pg.4) that R-FCN is finetuned with LR=1e-3 for 20k minibatches and then LR=1e-4 for 10k on VOC. So that's a 20k30k schedule. However, this doesn't seem to match the prototxts included here, which are 80k110k. 

2. I wonder if you would have LR schedule recommendations for other training sets, eg. just VOC'07. Of course it is easy to try with your code, but thought I should ask first, as this information could be helpful for many people :)

Thanks in advance!
